### PR TITLE
Updated categorical_hinge loss doc

### PR DESCRIPTION
**Pull request by lukasfolle**
This is just another minor update of the keras documentation.
Please feel free to make suggestions.

### CODE REVIEW

File: tensorflow/python/keras/losses.py [hunk 1]
```diff
@@ -836,7 +836,7 @@ def hinge(y_true, y_pred):
 
   Args:
     y_true: The ground truth values. `y_true` values are expected to be -1 or 1.
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: be converted

File: tensorflow/python/keras/losses.py [hunk 2]
```diff
@@ -850,6 +850,16 @@ def hinge(y_true, y_pred):
 
 @keras_export('keras.losses.categorical_hinge')
 def categorical_hinge(y_true, y_pred):
+  """Computes the categorical hinge loss between `y_true` and `y_pred`.
+
+      Args:
+        y_true: The ground truth values. `y_true` values are expected to be -1 or 1.
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: converted.

File: tensorflow/python/keras/losses.py [hunk 3]
```diff
@@ -850,6 +850,16 @@ def hinge(y_true, y_pred):
 
 @keras_export('keras.losses.categorical_hinge')
 def categorical_hinge(y_true, y_pred):
+  """Computes the categorical hinge loss between `y_true` and `y_pred`.
+
+      Args:
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: Remove space and align args with rest of the doc string.

**Comment by lufol** [comment]
@pavithrasv Should be fine now.

File: tensorflow/python/keras/losses.py (hunk 1 above)
Comment (pavithrasv) [reply tensorflow/python/keras/losses.py]: If binary (0 or 1) labels are provided they will be converted to -1 or 1

Comment (lufol) [reply tensorflow/python/keras/losses.py]: This is already in the docs.

Comment (pavithrasv) [reply tensorflow/python/keras/losses.py]: I meant in the right hand side in the line you have changed, i did not want to create another comment for this so commented on the prev one.

File: tensorflow/python/keras/losses.py [hunk 4]
```diff
@@ -836,7 +836,7 @@ def hinge(y_true, y_pred):
 
   Args:
     y_true: The ground truth values. `y_true` values are expected to be -1 or 1.
-      If binary (0 or 1) labels are provided we will convert them to -1 or 1.
+      If binary (0 or 1) labels are provided they will converted to -1 or 1.
```
Comment (pavithrasv) [inline tensorflow/python/keras/losses.py]: If binary (0 or 1) labels are provided they will be converted to -1 or 1.

Comment (lufol) [reply tensorflow/python/keras/losses.py]: Thanks @pavithrasv, fixed in b60b794.

**Review by pavithrasv — APPROVED** [review]
